### ISSUE TITLE
[Accessibility] Slides Page: 'Upload from URL' link lacks visual focus indicator

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#1018]
**Describe the bug**
The 'Upload from URL' and 'Upload from local' links in the slide upload modal are focusable but lack a visual indicator when they receive focus.

**To Reproduce**
1. Go to 'Slides'
2. Click on 'Upload'
3. Press the tab key to highlight all interactive elements
4. You'll notice that other elements have a visual focus indicator except the 'upload from url' link.

**Expected behavior**
Interactive elements should not only be keyboard accessible but also have a visual focus indicator.

**Screenshots**
![image](https://github.com/user-attachments/assets/c9b5cad9-53b6-475d-b029-e08405428844)

![image](https://github.com/user-attachments/assets/7f80952d-3e46-4293-8280-d15e2e0cd387)

**Comment by DAAS2** [comment]
Hi, could you please assign this issue to me so I can contribute and fix the bug.

Thanks

**Comment by DAAS2** [comment]
Hi @nwanduka 

I'd like to work on this issue! Could you please assign it to me?
Thanks.

**Comment by birm** [comment]
Hi @DAAS2 we don't assign issues usually, but if you want to work on something that you've identified, open a PR!

**Comment by rahulptl165** [comment]
Hi, @birm is this issue still opened, i want to work on this .

**Comment by rajku-dev** [comment]
Hi, @birm I have created a pull request fixing this bug.
